components/airbrake/sources/new-error-occurred/new-error-occurred.mjs

```diff
@@ -0,0 +1,78 @@
+import airbrake from "../../airbrake.app.mjs";
+
+export default {
+  name: "New Error Occurred",
+  version: "0.0.1",
+  key: "airbrake-new-error-occurred",
+  description: "Emit new event for each error occurred.",
```

Comment:
Can you add the link to the API docs here please?

Reply:
Normally we don't add docs link in sources, but I can.